packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -4,4 +4,225 @@ slug: /for-theme-developers
+</figure>
+<!-- /wp:image -->
+```
+
+:::
+
+### `setSiteOptions`
+
+With [`setSiteOptions`](/blueprints/steps#setSiteOptions) step, you can set [site options](https://developer.wordpress.org/apis/options/#available-options-by-category) such as site name, description or which page to use for posts.
+
+```json
+"steps": [
+	...,
+	{
+		"step": "setSiteOptions",
+		"options": {
+			"blogname": "Rich Tabor",
+			"blogdescription": "Multidisciplinary maker specializing in the intersection of product, design and engineering. Making WordPress.",
+			"show_on_front": "page",
+			"page_on_front": 6,
+			"page_for_posts": 2
+		}
+	},
+	...
+]
+```
+
+There's also a shorthand [`siteOptions`](/blueprints/steps/shorthands#siteoptions) than can be used instead of the `setSiteOptions` step.
+
+### `plugins`
+
+With the shorthand [`plugins`](/blueprints/steps/shorthands#plugins) you can set a list of plugins you want to be installed and activated with your theme in the Playground instance.
+
+```json
+"plugins": ["todo-list-block", "markdown-comment-block"]
+```
+
+You can also use the [`installPlugin`](/blueprints/steps#installPlugin) step to install and activate plugins for your Playground instance but the shorthand way is recommended
+
+### `login`
+
+With the shorthand [`login`](/blueprints/steps/shorthands#login) you can launch your Playground instance with the user logged
+
+```json
+ "login": true,
+```
+
+You can also use the [`login`](/blueprints/steps#login) step to launch your Playground instance logged in with any specific user.
+
+:::tip
+
+Some useful tools and resources provided by the Playground project to work with blueprints are:
+
+-   Check the [Blueprints Gallery](https://github.com/WordPress/blueprints/blob/trunk/GALLERY.md) to explore real-world code examples of using WordPress Playground to launch a WordPress site with a variety of setups.
+-   The [WordPress Playground Step Library](https://akirk.github.io/playground-step-library/#) tool provides a visual interface to drag or click the steps to create a blueprint for WordPress Playground. You can also create your own steps!
+-   The [WordPress Playground Step Library](https://akirk.github.io/playground-step-library/#) tool provides a visual interface to drag or click the steps to create a blueprint for WordPress Playground. You can also create your own steps!
+-   The [Blueprints builder](https://playground.wordpress.net/builder/builder.html) tool allows you edit your blueprint online and run it directly in a Playground instance.
+
+:::
+
+## Theme development
+
+### Local theme development and testing with Playground
+
+From the root folder of a block theme's code, you can quickly load locally a Playground instance with that theme loaded and activated by launching the [`wp-now` command](/developers/local-development/wp-now) or the [Visual Code Studio extension](/developers/local-development/vscode-extension).
```

Comment:
It's not clear where the reader would put the following code. To this point, they've been working in JSON files. It should be clear that this step would use a command line program. Keep in mind that many theme authors are going to be less familiar with this.

Something like this would work:

"...by launcing the `wp-now` command from your preferred command line program.

Reply:
I tweaked the sentence into:

> From the root folder of a block theme's code, you can quickly load locally a Playground instance with that theme loaded and activated. You can do that by launching the [`wp-now` command](/developers/local-development/wp-now) from your preferred command line program or the [Visual Code Studio extension](/developers/local-development/vscode-extension) from the [Visual Studio Code](https://code.visualstudio.com/) IDE.